ui/webui/src/components/storage/HelpAutopartOptions.jsx

```diff
@@ -1,42 +1,9 @@
-import React from "react";
 import cockpit from "cockpit";
 
-import {
-    Text,
-    TextContent,
-    TextVariants,
-} from "@patternfly/react-core";
-
 const _ = cockpit.gettext;
 
-export const helpEraseAll = (
-    <TextContent>
-        <Text component={TextVariants.p}>
-            {_("Removes all partitions on the selected devices, " +
-            "which includes partitions created by other operating systems.")}
-        </Text>
-
-        <Text component={TextVariants.p}>
-            {_("This option removes data from the selected devices, " +
-            "so make sure you have backed up your data.")}
-        </Text>
-    </TextContent>
-);
+export const helpEraseAll = _("Remove all partitions on the selected devices, including existing operating systems. Make sure you have backed up your data.");
 
-export const helpUseFreeSpace = (
-    <TextContent>
-        <Text component={TextVariants.p}>
-            {_("Retains your current data and partitions and uses " +
-            "only the unpartitioned space on the selected devices, " +
-            "assuming you have enough free space available.")}
-        </Text>
-    </TextContent>
-);
+export const helpUseFreeSpace = _("Keeps current disk layout and uses only available space.");
 
-export const helpCustomMountPoint = (
-    <TextContent>
-        <Text component={TextVariants.p}>
-            {_("This option requires that the selected device has formated partitions.")}
-        </Text>
-    </TextContent>
-);
+export const helpCustomMountPoint = _("This option requires that the selected device has formated partitions.");
```

Comment:
This is not related to this PR, just a note. These help texts evolved to be quite inconsistent ("Remove...", "Keeps..."), especially this one - the original intent was to explain what the option does (something like builtin help / documentation), which is also why it was moved to the drawer panel so that a bit more detailed help does not pollute the choices space too much). But this is for later updates, definitely not for this PR.